torch/_inductor/lowering.py

```diff
@@ -164,6 +164,8 @@ def maybe_layout_constraints(fn: Callable[..., Any]) -> Optional[Callable[..., A
 def tag_to_layout_constraint(tag):
     if tag == torch._C.Tag.needs_exact_strides:
         return constrain_to_fake_tensors
+    if tag == torch._C.Tag.needs_contiguous_strides:
+        return require_contiguous
```

Comment:
Can we construct the strides manually ? I had a [pr](https://github.com/pytorch/pytorch/pull/148235) to fix this but it was blocked by mkldnn tests. @CaoE has a pr to update https://github.com/pytorch/pytorch/pull/148424.  cc @CaoE @leslie-fang-intel we really need to land that.